### ISSUE TITLE
Dataviz styles failing when building with Bootstrap 4.3.1 and Kendo Boostrap 4.2.0

### ISSUE THREAD

**Issue by jcastillo-ait-asia** [telerik/kendo-themes#680]
## I'm submitting a...

* Bug report

## Current behavior

Module build failed at

```
ERROR in ./src/styles.scss
Module build failed (from /media/user/external/git/uber-spa/node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleBuildError: Module build failed (from /media/user/external/git/uber-spa/node_modules/sass-loader/lib/loader.js):

            top: calc( #{($padding-y-sm*2) + ($line-height*$font-size)} + 1px);
                                                                             ^
      Incompatible units: 'px' and 'rem'.
      in /media/user/external/git/uber-spa/node_modules/@progress/kendo-theme-bootstrap/modules/@progress/kendo-theme-default/scss/dataviz/_layout.scss (line 350, column 79)
    at runLoaders (/media/user/external/git/uber-spa/node_modules/webpack/lib/NormalModule.js:301:20)
    at /media/user/external/git/uber-spa/node_modules/loader-runner/lib/LoaderRunner.js:367:11
    at /media/user/external/git/uber-spa/node_modules/loader-runner/lib/LoaderRunner.js:233:18
```

## Environment
```
Angular CLI: 7.3.7
Node: 11.13.0
OS: linux x64
Angular: 7.2.11
... animations, common, compiler, compiler-cli, core, elements
... forms, http, language-service, platform-browser
... platform-browser-dynamic, router, service-worker

Package                            Version
------------------------------------------------------------
@angular-devkit/architect          0.13.7
@angular-devkit/build-angular      0.13.7
@angular-devkit/build-ng-packagr   0.13.7
@angular-devkit/build-optimizer    0.13.7
@angular-devkit/build-webpack      0.13.7
@angular-devkit/core               7.3.7
@angular-devkit/schematics         7.3.7
@angular/cdk                       7.3.6
@angular/cli                       7.3.7
@angular/flex-layout               7.0.0-beta.24
@angular/material                  7.3.6
@angular/pwa                       0.13.7
@ngtools/json-schema               1.1.0
@ngtools/webpack                   7.3.7
@schematics/angular                7.3.7
@schematics/update                 0.13.7
ng-packagr                         4.7.1
rxjs                               6.4.0
typescript                         3.2.4
webpack                            4.29.0
```

Kendo dependencies:
```
"bootstrap": "4.3.1",
"@progress/kendo-angular-buttons": "4.3.3",
"@progress/kendo-angular-charts": "3.8.0",
"@progress/kendo-angular-dateinputs": "3.7.1",
"@progress/kendo-angular-dialog": "3.11.0",
"@progress/kendo-angular-dropdowns": "3.5.2",
"@progress/kendo-angular-excel-export": "2.3.0",
"@progress/kendo-angular-grid": "3.13.1",
"@progress/kendo-angular-inputs": "4.2.1",
"@progress/kendo-angular-intl": "1.7.0",
"@progress/kendo-angular-l10n": "1.3.0",
"@progress/kendo-angular-layout": "3.3.0",
"@progress/kendo-angular-menu": "1.2.0",
"@progress/kendo-angular-notification": "1.2.0",
"@progress/kendo-angular-popup": "2.5.0",
"@progress/kendo-angular-resize-sensor": "3.1.2",
"@progress/kendo-angular-scheduler": "0.10.1",
"@progress/kendo-angular-sortable": "2.0.1",
"@progress/kendo-angular-upload": "4.2.2",
"@progress/kendo-data-query": "1.5.1",
"@progress/kendo-date-math": "1.4.0",
"@progress/kendo-drawing": "1.5.9",
"@progress/kendo-recurrence": "0.1.2",
"@progress/kendo-theme-bootstrap": "3.2.3",
```

**Comment by jcastillo-ait-asia** [comment]
Bumping on this issue. Can this be replicated at your end?

**Comment by jcastillo-ait-asia** [comment]
I wonder if there's anything that can be done for this one...

**Comment by jcastillo-ait-asia** [comment]
Dataviz scss is stilling failing even after upgrading to kendo-theme-bootstrap 3.4.1

```
“@progress/kendo-angular-buttons”: “4.4.1",
“@progress/kendo-angular-charts”: “3.9.0",
“@progress/kendo-angular-dateinputs”: “3.7.3",
“@progress/kendo-angular-dialog”: “3.12.0",
“@progress/kendo-angular-dropdowns”: “3.5.4",
“@progress/kendo-angular-excel-export”: “2.3.0",
“@progress/kendo-angular-grid”: “3.14.2",
“@progress/kendo-angular-inputs”: “5.0.0",
“@progress/kendo-angular-intl”: “1.7.0",
“@progress/kendo-angular-l10n”: “1.4.0",
“@progress/kendo-angular-layout”: “3.3.1",
“@progress/kendo-angular-menu”: “1.2.0",
“@progress/kendo-angular-notification”: “1.2.0",
“@progress/kendo-angular-popup”: “2.5.0",
“@progress/kendo-angular-resize-sensor”: “3.1.2",
“@progress/kendo-angular-scheduler”: “0.12.1",
“@progress/kendo-angular-sortable”: “2.1.0",
“@progress/kendo-angular-upload”: “4.3.1",
“@progress/kendo-data-query”: “1.5.1",
“@progress/kendo-date-math”: “1.4.0",
“@progress/kendo-drawing”: “1.5.12",
“@progress/kendo-recurrence”: “0.1.2",
“@progress/kendo-theme-bootstrap”: “3.4.1",
```

**Comment by jcastillo-ait-asia** [comment]
Hi. Any updates on this one? I can see It's still an issue in 4.0.0

**Comment by jcastillo-ait-asia** [comment]
Error still persisting and not getting feedback from you guys. What's going on?

```
ERROR in ./src/styles.scss
Module build failed (from /Users/crash/git/project/project-spa/node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleBuildError: Module build failed (from /Users/crash/git/project/project-spa/node_modules/sass-loader/lib/loader.js):
           top: calc( #{($padding-y-sm*2) + ($line-height*$font-size)} + 1px);
                       ^
     Incompatible units px and rem.
   ╷
350 │             top: calc( #{($padding-y-sm*2) + ($line-height*$font-size)} + 1px);
   │                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   ╵
 node_modules/@progress/kendo-theme-bootstrap/modules/@progress/kendo-theme-default/scss/dataviz/_layout.scss 350:26         @content
 node_modules/@progress/kendo-theme-bootstrap/modules/@progress/kendo-theme-default/scss/mixins/core/_import-once.scss 12:9  exports()
 node_modules/@progress/kendo-theme-bootstrap/modules/@progress/kendo-theme-default/scss/dataviz/_layout.scss 284:1          @import
```

Environment now is:

```
    "@progress/kendo-angular-buttons": "5.0.0",
    "@progress/kendo-angular-charts": "4.1.0",
    "@progress/kendo-angular-common": "1.1.0",
    "@progress/kendo-angular-dateinputs": "4.0.1",
    "@progress/kendo-angular-dialog": "4.1.0",
    "@progress/kendo-angular-dropdowns": "4.0.0",
    "@progress/kendo-angular-excel-export": "3.0.1",
    "@progress/kendo-angular-grid": "4.2.1",
    "@progress/kendo-angular-inputs": "6.1.1",
    "@progress/kendo-angular-intl": "2.0.0",
    "@progress/kendo-angular-l10n": "2.0.0",
    "@progress/kendo-angular-layout": "4.0.0",
    "@progress/kendo-angular-menu": "2.0.0",
    "@progress/kendo-angular-notification": "2.0.0",
    "@progress/kendo-angular-pdf-export": "2.0.0",
    "@progress/kendo-angular-popup": "3.0.1",
    "@progress/kendo-angular-scheduler": "0.16.0",
    "@progress/kendo-angular-sortable": "3.0.1",
    "@progress/kendo-angular-upload": "5.1.0",
    "@progress/kendo-data-query": "1.5.2",
    "@progress/kendo-date-math": "1.4.1",
    "@progress/kendo-drawing": "1.5.12",
    "@progress/kendo-recurrence": "1.0.0",
    "@progress/kendo-theme-bootstrap": "4.2.0",
    "@progress/kendo-theme-material": "3.2.0",
```

**Comment by feitzi** [comment]
Same here! Kendo team! this has to be fixed!! 
I have the problem if I want to use kendo with font-size  REM. Like here telerik/kendo-themes#463

**Comment by joneff** [comment]
Fixed by #1060 courtesy of @feitzi
